libraries/rush-sdk/package.json

```diff
@@ -8,11 +8,11 @@
     "directory": "apps/rush-sdk"
   },
   "homepage": "https://rushjs.io",
-  "main": "lib/index.js",
+  "main": "lib-shim/index.js",
   "typings": "dist/rush-lib.d.ts",
   "scripts": {
-    "build": "heft build --clean",
-    "_phase:build": "heft build --clean",
+    "build": "heft build --clean && node lib-shim/generate-stubs",
+    "_phase:build": "heft build --clean && node lib-shim/generate-stubs",
```

Comment:
Use a Heft run script eventAction in `heft.json`?

Reply:
Oh did we implement that feature? It's completely undocumented 😓

@iclanton @D4N14L

Reply:
Fixed